app/controllers/api/messaging.php

```diff
@@ -106,20 +105,18 @@
     ->label('sdk.response.model', Response::MODEL_PROVIDER)
     ->param('providerId', '', new CustomId(), 'Provider ID. Choose a custom ID or generate a random ID with `ID.unique()`. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.')
     ->param('name', '', new Text(128), 'Provider name.')
-    ->param('from', '', new Text(256), 'Sender Email Address.')
+    ->param('from', '', new Text(256), 'Sender email address.')
     ->param('apiKey', '', new Text(0), 'Sendgrid API key.')
-    ->param('default', false, new Boolean(), 'Set as default provider.', true)
     ->param('enabled', true, new Boolean(), 'Set as enabled.', true)
     ->inject('dbForProject')
     ->inject('response')
-    ->action(function (string $providerId, string $name, string $from, string $apiKey, bool $default, bool $enabled, Database $dbForProject, Response $response) {
+    ->action(function (string $providerId, string $name, string $from, string $apiKey, bool $enabled, Database $dbForProject, Response $response) {
         $providerId = $providerId == 'unique()' ? ID::unique() : $providerId;
         $provider = new Document([
             '$id' => $providerId,
             'name' => $name,
             'provider' => 'sendgrid',
             'type' => 'email',
-            'default' => $default,
```

Comment:
Let's keep it here but set to false

Reply:
no need to keep this, default value is false